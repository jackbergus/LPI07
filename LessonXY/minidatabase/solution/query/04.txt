lOaD table/test.txt
project A,C
cOUNT
PRiNT
group C by C with * AS d
PRINT