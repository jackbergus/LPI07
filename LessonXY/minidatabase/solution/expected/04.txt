7
A,C
1,3
1,4
4,6
4,7
5,7
7,9
7,23
C,d
3,3
4,4
6,6
7,49
9,9
23,23
